src/platform/wasm.zig

```diff
@@ -53,6 +56,20 @@ pub const Platform = struct {
                 .width = js.machCanvasGetFramebufferWidth(id),
                 .height = js.machCanvasGetFramebufferHeight(id),
             },
+
+            // TODO initialize these properly
```

Comment:
What does this mean? Initialize them to their current state? How hard is that to do?

Reply:
I can add code to the js side to do this and plumb it through, but I'm reluctant to make non-trivial changes to the js because none of the wasm examples work for me so I have no way to test whether I'm writing gibberish or not.

Reply:
Got it, let's do that in a follow-up PR.